BibTeX.js

```diff
@@ -262,7 +263,14 @@ function setKeywordDelimRe( val, flags ) {
 function processField(item, field, value, rawValue) {
 	if(Zotero.Utilities.trim(value) == '') return null;
 	if(fieldMap[field]) {
-		item[fieldMap[field]] = value;
+		//map DOIs + Label to Extra for unsupported item types
```

Comment:
Ah I see. I figured it'd be better if the item _isn't_ mapped to item.DOI when it's written to Extra, so that we don't get DOI twice when using bibtex from a web translator that also adds DOIs (I know a couple do). What's the advantage of doing this your way around?